_posts/2020-09-16-cockpit-228.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Cockpit 228
+author: kkoukiou
+date: 2020-09-16
+tags: cockpit linux
+slug: cockpit-228
+category: release
+summary: Cockpit
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly. Here are the release notes from Cockpit version 228.
+
+### Accounts: Allow setting weak passwords
+
+Cockpit now allows users to set up a weak password. The user gets notified that
+the password is weak, but clicking the submit button again will use the selected password anyway.
+This is similar to how weak passwords are handled in the Anaconda installer.
+
+![Accounts weak password](/images/accounts-weak-password.png){:.screenshot}
+
+### Changes to remote host logins
+
+Cockpit now never tries to log into remote hosts with your initial login password,
```

Comment:
I think it is too easy to miss the "never" here and read this is as "Cockpit now tries to log into remote hosts with your initial login password".

What about:

Cockpit used to try to log into remote hosts with your initial login password when "Reuse my password for remote connections" was selected during login. This option has been removed and consequently ...

Reply:
I reworked it accordingly.